### ISSUE TITLE
`ByName` doesn't handle input as a child of a label

### ISSUE THREAD

**Issue by stroiman** [gost-dom/shaman#1]
The case, where an input is embedded in a label doesn't find the input

```html
<label>First name: <input type="text" name="first-name" /></label
```

The accessibility name for the input field should be, "First name: "
